lib/config/app_config.dart

```diff
@@ -0,0 +1,5 @@
+class AppConfig {
```

Comment:
Duplicate/Unused `AppConfig` class; see comment in `lib/views/utils/routes.dart`.

Reply:
Yes make sense.
Done ✅